Intel_SighPP_CMake/Intel_SighPP_CMake/src/interface_controller/api_controller.hpp

```diff
@@ -19,10 +19,6 @@ class ApiController
 	/// </summary>
 	/// <param name="user">User to add to current users.</param>
 	void add_user(ApiUserInterface& user);
+	void new_items(const std::vector<ClassificationItem>& results);
```

Comment:
Was there a reason for removing the documentation of this method?

Reply:
it may have disappeared in the merge

Reply:
Fixed.